### DISCUSSION TITLE
UFS Repost: ERROR IN SRW POST-PROCESSING, WHEN ${FHR} IS FOR EXAMPLE 101

### DISCUSSION THREAD

**Discussion by ShawnCebulaNOAA** [ufs-community/ufs-srweather-app#553] [Q&A] [answered]
There was a question over on the UFS Community Forum I've grabbed and pasted down below from user liviu.oana88.  I've asked her to come over here and provided a link for this discussion post for her to check out.

Shawn Cebula, EPIC End User Support

--copied post below--

Hello, 

 

I am running a SRW case (v. 2.1.0). However, after the simulation finishes (a month of forecast), I have 744 output files, and when I run ./run_post.sh, after hour 099, post fails with the following error:

 

FATAL ERROR:
ERROR:
  From script:  "exregional_run_post.sh"
  Full path to script:  "/home/wrf/fv3_latest/ufs_intel/ufs-srweather-app/scripts/exregional_run_post.sh"
The ${fhr} variable contains a 3-digit integer whose first digit is not
0.  In this case, it is not clear how to set the variable post_fhr used
in constructing the grib2 file names generated by UPP:
  fhr = "657"
Exiting with nonzero status.
End exregional_run_post.sh at Fri 20 Jan 2023 04:01:45 PM UTC with error code 1 (time elapsed: 00:00:02)
FATAL ERROR:
ERROR:
  From script:  "JREGIONAL_RUN_POST"
  Full path to script:  "/home/wrf/fv3_latest/ufs_intel/ufs-srweather-app/jobs/JREGIONAL_RUN_POST"
Call to ex-script corresponding to J-job "JREGIONAL_RUN_POST" failed.
Exiting with nonzero status.
End JREGIONAL_RUN_POST at Fri 20 Jan 2023 04:01:45 PM UTC with error code 1 (time elapsed: 00:00:03)
 

 

 Is there a way I can fix this, since not all simulations are less than 4 days in lenght?

 

Thank you!

**Comment by LiviuOana** [comment, marked as answer]
Hello, I am the original poster of this issue.

I think I managed to fix this, by modifying the "ufs-srweather-app/scripts/exregional_run_post.sh" script, around the line  245, in the "else case" by just setting  `post_fhr=${fhr}`. I tested and UPP processes those files too.

**Comment by ShawnCebulaNOAA** [comment]
Thank you Liviu, glad to see you were able to fix it!  I'll mark this as answered then!

Shawn Cebula, EPIC End User Support
